client/src/components/Tag.js

```diff
@@ -0,0 +1,17 @@
+import styled from "@emotion/styled";
+
+const Tag = styled.button`
+  font-family: "SFUIreg";
+  font-size: 1em;
+  color: var(--font-semi-dark);
+  border: 0;
+  border-radius: 3em;
+  padding: 1em 2em;
+  margin: 2em 0em 0.5em;
+  display: inline-block;
+  line-height: 1;
+  background: var(--tag-yellow);
```

Comment:
There is an issue if you have the name of the color in your variable name. You can not create a theme with different colors :). 
Instead, use names which describe the use case or function.
See https://github.com/manuelsanchez2/doicheliving-app/pull/35#discussion_r490178649

Reply:
True that. 🤦